server/protocols/jmap/src/main/java/org/apache/james/jmap/model/mailbox/Rights.java

```diff
@@ -206,7 +211,7 @@ public Rights removeEntriesFor(Username username) {
             rights.asMap()
                 .entrySet()
                 .stream()
-                .filter(entry -> entry.getKey().equals(username))
+                .filter(entry -> !entry.getKey().equals(username))
```

Comment:
then how does the previous unit tests work?

Reply:
It did not it's just that this should be in the previous commit, therefore I did a fixup for this commit to remove this diff from this commit and an other fixup so this diff get included in the previous commit